bundles/net.tourbook.device.garmin.fit/src/net/tourbook/device/garmin/fit/listeners/MesgListener_Record.java

```diff
@@ -375,6 +378,9 @@ private void setRecord_DeveloperData(final RecordMesg mesg, final TimeData timeD
                }
             }
 
+            timeData.powerDataSource = devField.getName().equals(DEV_FIELD_NAME__POWER) ? Messages.PrefPage_Fit_Combo_Power_Data_Source_Stryd
```

Comment:
timeData.powerDataSource should contain static text and not text which is translated that later on this field could be used for any kind of reports/statistics/views

Reply:
Resolved in upcoming PR